### PR TITLE
229 - add sleep to avoid hitting rate limit

### PR DESCRIPTION
**Pull request by ijadams**
https://github.com/revelrylabs/reusable-stories/issues/229

### CODE REVIEW

File: slax.iml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lukeledet) [inline slax.iml:1]: Add this to the gitignore

File: lib/slax/commands/github_commands.ex [hunk 2]
```diff
@@ -48,6 +48,8 @@ defmodule Slax.Commands.GithubCommands do
       ) do
     repo = "#{org_name}/#{project_name}"
 
+    sleep(1.5)
```
Comment (lukeledet) [inline lib/slax/commands/github_commands.ex:51]: CI is failing because this function doesn't exist

**Review by lukeledet — CHANGES_REQUESTED** [review]

File: lib/slax/commands/github_commands.ex (hunk 2 above)
Comment (ijadams) [reply lib/slax/commands/github_commands.ex]: My bad no idea why I thought this repo was ruby lol.

**Review by lukeledet — APPROVED** [review]
👍👍

**Review by CurtisVonRubenhoff — APPROVED** [review]
seems reasonable to me.

**Pull request merged**